Open external links in new tab - Closes #522 (#541)
LiskHQ/lisk-docs: modules/ROOT/pages/update/source.adoc

<1> Navigate into the root directory of Lisk Core.
<2> Fetch the latest releases.

> <3> Check out latest release tag, please see {url_core_releases}[core releases^].

Reviewer: ```suggestion
<3> To view the latest release tag, please see {url_core_releases}[core releases^].
```
